tests/trainers/tensorflow_examples.py

```diff
@@ -0,0 +1,53 @@
+"""Basic tests for the Python 3.6 and 3.9 base container images,based on tensorflow."""
+import tensorflow as tf
+
+
+def tensorflow_example_1():
+
+    print("TensorFlow version:", tf.__version__)
+
+    mnist = tf.keras.datasets.mnist
+
+    (x_train, y_train), (x_test, y_test) = mnist.load_data()
+    x_train, x_test = x_train / 255.0, x_test / 255.0
+
+    model = tf.keras.models.Sequential(
+        [
+            tf.keras.layers.Flatten(input_shape=(28, 28)),
+            tf.keras.layers.Dense(128, activation="relu"),
+            tf.keras.layers.Dropout(0.2),
+            tf.keras.layers.Dense(10),
+        ]
+    )
+
+    predictions = model(x_train[:1]).numpy()
+
+    predictions
+
+    print(tf.nn.softmax(predictions).numpy())
```

Comment:
Why do you print this out? It's never checked anywhere by the tests (same for the following three print statements)